azure-pipelines.yml

```diff
@@ -94,3 +99,18 @@ jobs:
           testResultsFiles: '$(testResultsFiles)'
           testRunTitle: 'Windows - $(name)'
           mergeTestResults: true
+  - job: GitHub_Release
+    displayName: Release to GitHub
+    dependsOn: 
+    - Windows
+    - Linux
+    condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v1.'))  
+    steps:
+    - task: GitHubRelease@0
+      displayName: 'GitHub release (create)'
+      inputs:
+        gitHubConnection: 'github-zenithworks'
+        title: JupyterLab Release version $(Build.SourceBranchName) 
+        releaseNotesSource: input
+        releaseNotes: |         
+          [Installation](https://github.com/zenithworks/jupyterlab/blob/master/README.md#installation)   |   [Getting Started](https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html)   |   [Changelog](https://jupyterlab.readthedocs.io/en/latest/getting_started/changelog.html)     
```

Comment:
This just links to the changelog, but the example you note has a list of merge commits. How is GitHub seeing that entire list of merge commits, and where are these links going?

Also, presumably the installation link needs to point to the official repo.

Reply:
> where are these links going

Ah, I see the links at the top of the release notes. Where is the list of merge commits coming from, though?

Reply:
My guess is that they're auto-populated from a delta between tags.  We could replace it with our own for the larger releases.

Reply:
@blink1073 is right. its auto populated from delta between this release and last published release. If required this auto population can be switched off too